Common/Core/RecoDecay.h

```diff
@@ -598,11 +596,11 @@ class RecoDecay
       //printf("Stage %d: Mother PDG: %d\n", stage, PDGParticleIMother);
       if (PDGParticleIMother == PDGMother) { // exact PDG match
         sgn = 1;
-        indexMother = indexMotherTmp;
+        indexMother = particleMotherIdx;
```

Comment:
Revert. Variables starting with `particle` are meant to be particles everywhere else in the code.